frontend/src/diagram/sprotty/utils/geometry.tsx

```diff
@@ -0,0 +1,111 @@
+/*******************************************************************************
+ * Copyright (c) 2022 THALES GLOBAL SERVICES.
```

Comment:
Why THALES in the copyright?

Reply:
Using the auto-completion after typing `epl` initializes the header file.

Reply:
done